front/components/assistant/conversation/content_creation/ClientExecutableRenderer.tsx

```diff
@@ -316,9 +317,84 @@ export function ClientExecutableRenderer({
               isInDrawer={true}
               ref={iframeRef}
             />
+            <PreviewActionButtons
+              owner={owner}
+              conversation={conversation}
+              lastEditedByAgentConfigurationId={
+                lastEditedByAgentConfigurationId
+              }
+              onRevert={onRevert}
+              isFullScreen={isFullScreen}
+              exitFullScreen={exitFullScreen}
+              enterFullScreen={enterFullScreen}
+              reloadFile={reloadFile}
+            />
           </div>
         )}
       </div>
     </div>
   );
 }
+
+interface PreviewActionButtonsProps {
+  owner: LightWorkspaceType;
+  conversation: ConversationWithoutContentType;
+  lastEditedByAgentConfigurationId?: string;
+  onRevert: () => void;
+  isFullScreen: boolean;
+  enterFullScreen: () => void;
+  exitFullScreen: () => void;
+  reloadFile: () => void;
+}
+
+function PreviewActionButtons({
+  lastEditedByAgentConfigurationId,
+  onRevert,
+  isFullScreen,
+  enterFullScreen,
+  exitFullScreen,
+  reloadFile,
+}: PreviewActionButtonsProps) {
+  return (
+    <div className="fixed bottom-4 right-3 flex flex-col gap-1 rounded-lg bg-white p-1 shadow-md dark:bg-gray-900">
+      {lastEditedByAgentConfigurationId && (
+        <Tooltip
```

Comment:
I cannot adjust the tooltip position with button component, and since this is very one off I didn't want to update the button component, since I'm using Tooptip here